python/ray/tests/test_memstat.py

```diff
@@ -75,17 +87,16 @@ def f(y):
     assert num_objects(info) == 4, info
     # Task argument plus task return ids.
     assert count(info, TASK_CALL_OBJ) == 2, info
-    assert count(info, DRIVER_PID) == 1, info
-    assert count(info, WORKER_PID) == 1, info
+    assert count(info, DRIVER_PID) == 2, info
```

Comment:
Why is it changed? (We shouldn't have any changing values to the test.)

Reply:
Right, I changed it here because the original memory_summary() stub groups by driver and worker, so it only shows those strings once. However, this new implementation specifies for each object whether it's within the driver or worker. The logic is still the same, just the way the string is formatted is different. I can group by Driver and Worker as well, however, if that is needed.